README.md

```diff
@@ -39,3 +39,12 @@ For Day 1 cutover, once we have main branch:
 - Update `googleapis_commit.txt` to an appropriate value
 - Update `.github/workflows/googleapis_hermetic_sync.yaml` to point to
   the main branch.
+
+### The BOM coverage and its version
+
+Review the artifact name "google-cloud-gapic-bom" in the bom directory and
+configure the version managed by Release Please. Ensure the BOM is part of the
+entire release pipeline.
```

Comment:
@ddixit14 This message is for you in near future.

Reply:
Sure, will keep this in mind.